### DISCUSSION TITLE
Can't receive email verification for Github student pack

### DISCUSSION THREAD

**Discussion by ZakaryaMeddahi** [community/community#53475] [GitHub Education] [answered]
I'm trying to subscribe to the GitHub student pack, but I Can't receive email verification on it :(

**Comment by NJacksonDev** [comment, marked as answer]
Here are some steps which may be helpful - if they do not work, I would contact GitHub support for additional assistance.

- Check your spam or junk email folder: Sometimes, the verification email may be mistakenly marked as spam or junk by your email provider. Make sure to check those folders to see if the email was routed there.

- Verify that the email address is correct: Make sure that the email address you provided during the application process is accurate and correctly spelled. If there was a mistake, you may need to update your email address in your GitHub account settings.

- Resend the verification email: If you still haven't received the verification email, try clicking the "Resend verification email" button on the GitHub Student Pack application page. This will send another email to the same address.

**Reply by AbRahman-ra** [reply]
I tried all of that and it is still not working. I am facing the same issue.

**Reply by Magmaron25** [reply]
try opening outlook on the same mail it worked for me

**Reply by AbRahman-ra** [reply]
I tried now, still. I contacted the support, provided all the evidences I'm enrolled. And still...

**Reply by ifeoluwaayo** [reply]
It's really annoying cause my school has the same issue

**Reply by camerondodds-sv** [reply]
This is super annoying. I contacted support, copied and pasted my email in (to guarantee it's typed correctly), and checked spam/junk. The verification email still isn't coming in, and github support's employees don't have a technical background that allows them to figure out what's going on.

**Reply by n3rm4121** [reply]
Could you make sure you have checked in your school-issued email address, not in your personal email address? You will not get any email in your personal email connected to your school-issued email address. For example, if my personal email is a@a.com which I have given to my school, and the school has issued me an email as 111@xyz.edu.au then you have to look into 111@xyz.edu.au in Microsoft Outlook.

This should work!

**Reply by camerondodds-sv** [reply]
I definitely checked my school email and It's definitely not going through. That is a good thing to double check though

**Reply by camerondodds-sv** [reply]
Github support eventually checked their logs and told me that my school's email service accepted the email. I talked to my school and they blocked all incoming emails because I had email forwarding turned on. The school was worried that the email addresses that my emails got forwarded to wasn't secure, so they stopped the emails from arriving in the first place. They also didn't notify me of that fact. 

Long story short, I turned off message forwarding and everything worked. Hope that helps

**Reply by camerondodds-sv** [reply]
@phoriah @j-onty

**Reply by UsminDhakal** [reply]
I am having the same problem

**Comment by phoriah** [comment]


**Reply by j-onty** [reply]
same unable to fix

**Comment by j-onty** [comment]
> @phoriah @j-onty
Mine randomly fixed itself, I wasn't receiving any emails to my student email, so I couldn't get a verification code, but I tried again today, but this time connected to my university Wi-Fi and was suddenly able to receive emails and log in. Not sure if it will be the same for anyone else, tho. Thanks for the help tho, I will try that too just in case.

**Comment by RayyanNafees** [comment]
having the same problem for AMU, India
The @myamu.ac.in mails

**Reply by Joshuaosunlakin3433** [reply]
Same problem, the solution above didn't fix it
